packages/ui/src/machines/authMachine.ts

```diff
@@ -130,7 +130,7 @@ export const authMachine = createMachine<AuthContext, AuthEvent>(
           const actor = signUpActor.withContext({
             authAttributes: event.data?.authAttributes ?? {},
             intent: event.data?.intent,
-            formValues: {},
+            formValues: { country_code: '+1' },
```

Comment:
This is where I stuck the default `country_code` state. Would love some feedback on the best way to handle this.

Reply:
Honestly, this makes sense to me since we're defaulting the value for the view (vs. the view having the default itself on every platform).